Source/WebCore/en.lproj/Localizable.strings

```diff
@@ -175,6 +175,9 @@
 /* Allow button title in speech recognition prompt */
 "Allow (speechrecognition)" = "Allow";
 
+/* Allow */
```

Comment:
are these ok to add?

Reply:
This isn't being added, just moved. The script sorts alphabetically.

Though we should preserve the existing comment for this string, which was `Display USDZ file`.